examples/playground/src/configuration.js

```diff
@@ -38,8 +38,5 @@ export default {
   },
 
   // Constants that should be resolved with the provided values by JSON converter
-  constants: {
-    DracoLoader,
-    DracoWorkerLoader
-  }
+  constants: {}
```

Comment:
In this case, is constants still needed?

Reply:
Not really, but this is an example, so I'm leaving it for demo purpose.